docs/getting_started/configuration/options.rst

```diff
@@ -72,6 +72,7 @@ order to successfully access a third-party Reddit site:
     (default: ``t3_``).
 :subreddit_kind: The type prefix for subreddits on the :class:`.Reddit` instance
     (default: ``t5_``).
+:window_size: The number of seconds between rate limit resets (default: 600).
```

Comment:
I think this should be in a different section. This section is for configuring PRAW with a custom Reddit instance.

Reply:
Moved